completions/_vagrant

```diff
@@ -14,6 +14,13 @@ _vagrant () {
 		fi
 		_describe -t commands "vagrant $1" vms -V1
 	}
+	getboxes() {
+        	local -a boxes
+        	boxes=(
+			$(vagrant box list | cut -d " " -f 1)
+            	)
+	        _describe -t commands "vagrant init $2" boxes -V1
```

Comment:
I'm not following what $2 is in line 22, I just used that because other commands had similar...